src/modules/AuthModule/components/Register/Register.tsx

```diff
@@ -150,34 +149,31 @@ const appendToFormData=(data:FormValues)=>{
         )}
         </Stack>
          <Stack my="5px" mr="5px" >
-         <label htmlFor="country"  style={{ fontSize: "1.2rem", fontWeight: 300, opacity: 0.8 }}>Country</label>
+         <InputLabel htmlFor="country" style={{ color:'rgba(21, 44, 91, 1)',fontSize: "1.1rem", fontWeight: 400, marginBottom: 6}}>Country</InputLabel>
          <Stack>
         
          <TextField type="text" sx={{
                    
           marginBottom: 1,
-          bgcolor: "#f5f6f88f"
+          bgcolor: "rgba(245, 246, 248, 1)"
         }} id="country" placeholder="Please type here ..." {...register("country",{required:"Country is Required",minLenth:3,maxLength:10
          
         })}></TextField>
         </Stack>
         {errors.country && errors.country.type  && (
           <Typography color="error" sx={{
                      marginBottom: 1,
-                    bgcolor: "#f5f6f88f"
-                  }}>{errors.country.message}</Typography>
+                     bgcolor: "rgba(245, 246, 248, 1)"

```

Comment:
We should use the colors and spacing system in MUI as I mentioned before.  As Using the color system will make it easier for us to update our colors and implement dark/light mode without any effort.